README.rst

```diff
@@ -299,6 +299,19 @@ this, you have to pass in ``cssutils_logging_handler`` and
     >>> mylog.getvalue()
     'CSSStylesheet: Unknown @rule found. [2:1: @keyframes]\n'
 
+Choosing the cache implementation
+-----------------------------------
+
+By default, ``premailer`` uses `LFUCache
+<https://cachetools.readthedocs.io/en/latest/#cachetools.LRUCache>`__ to cache
```

Comment:
LFUCache or LRUCache?

Reply:
Ie. which is right, the link href or the link text?

Reply:
Judging by the code it's LRUCache

Reply:
Its LRU. Sorry for the confusion.